frontend/routes/auth/wallets/wallets.spec.tsx

```diff
@@ -56,15 +53,22 @@ describe('Wallets', () => {
     )
     // Wait for list to load
     await screen.findByTestId(locators.listItem)
-    expect(screen.getByTestId(walletsWalletName)).toHaveTextContent('wallet 1')
-    expect(screen.getByTestId(locators.copyWithCheck)).toHaveTextContent('07248a…3673')
+    expect(screen.getByTestId(walletLocators.walletsWalletName)).toHaveTextContent('wallet 1')
+    expect(screen.getByTestId(walletLocators.walletsExplorerLink)).toHaveTextContent('07248a…3673')
```

Comment:
I think we should check this is visible too. The E2E tests do not currently have a need to click this link so whether the element is visible to interact with is not covered there